### ISSUE TITLE
Docs are broken

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/poliastro#1229]
I think we broke the docs after I merged some of the recent pull requests: either #1184 or #1190, both by @Yash-10.

Here's the latest failed build:

https://readthedocs.org/projects/poliastro/builds/13776407/

```
UnitConversionError: Can only apply 'add' function to quantities with compatible dimensions
```

there's a units problem in the NEOs notebook.

**Comment by astrojuanlu** [comment]
Both Read the Docs and our `py38_online_test` check have failed, see for example https://github.com/poliastro/poliastro/runs/2594411686.

**Comment by jtegedor** [comment]
I found the same while doing something else, I do not think it is related specifically to those PR.

After doing the changes here https://github.com/poliastro/poliastro/pull/1228, docs and pytest work again.

**Comment by astrojuanlu** [comment]
Oh, now I know what's happening! The `from_sbdb` old code looked weird to me, but I remembered it had to do with some weird astroquery thing. I checked their release notes and saw that https://github.com/astropy/astroquery/pull/2011 has been released, hence the jplsbdb service now returns astropy quantities. It makes total sense to merge #1228, doing it now.

**Comment by astrojuanlu** [comment]
Closed in #1228 🚀

**Comment by astrojuanlu** [comment]
For future readers: Until we release poliastro 0.15.1 with a fix for this, a solution is to downgrade astroquery:

```
pip install "astroquery<0.4.2"
```
